.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
> So, I should change that part of package.json to something like this...
> ... and remove .npmignore?

Correct.

However, `./lib` should be `lib` AFAIK. I have already been fighting with npm on that.

What do you think? Should the untranpiled version be included in the npm package? (imho not)